source/declarations.tex

```diff
@@ -237,8 +237,9 @@
 A \grammarterm{nodeclspec-function-declaration} shall declare a
 constructor, destructor, or conversion function.
 \begin{note}
-A \grammarterm{nodeclspec-function-declaration} can only be used in a
-\grammarterm{template-declaration}\iref{temp.pre},
+Since a member function cannot be subject to a non-defining declaration
```

Comment:
Please use "Because" rather than "Since" for relationships that are causal rather than temporal (since can mean either, whereas because is unambiguous).

Reply:
@zygoloid: I made the requested change, PTAL?